internal/repos/import_test.go

```diff
@@ -248,3 +259,10 @@ func goListModulesStub(dir string) ([]byte, error) {
 }
 `), nil
 }
+
+func findRemoteStub(importpath string) (remote, vcsName string, err error) {
```

Comment:
See the comment below about `RemoteCache`. I think `newStubRemoteCache` should be used instead.